example/docker-compose.yml

```diff
@@ -1,26 +1,21 @@
-# Mongo DB instance
-# used for test purpose only
-# hosted to http://localhost:8081
-
-# Use root/example as user/password credentials
-version: '3.1'
+version: '3'
 
 services:
-
-  mongo:
-    image: mongo
-    restart: always
+  example-checker:
+    build: .
+    # coordinate the exposed port with other service/checker authors to avoid overlap
     ports:
-      - 27017:27017
+      - 7891:8000
     environment:
-      MONGO_INITDB_ROOT_USERNAME: root
-      MONGO_INITDB_ROOT_PASSWORD: example
-
-  mongo-express:
-    image: mongo-express
-    restart: always
-    ports:
-      - 8081:8081
+     - MONGO_ENABLED=1
+     - MONGO_HOST=example-db
+     - MONGO_PORT=27017
+     - MONGO_USER=enoislove
+     - MONGO_PASSWORD=enoislife
+  example-db:
+    image: mongo
+    volumes:
+      - ./db_data:/data/db
     environment:
-      ME_CONFIG_MONGODB_ADMINUSERNAME: root
-      ME_CONFIG_MONGODB_ADMINPASSWORD: example
\ No newline at end of file
+      MONGO_INITDB_ROOT_USERNAME: enoislove
```

Comment:
Do we get problems when we don't change these for the final ctf? Evil redirects to Mongo, when we use `requests` or sthm?=?

Reply:
I don't think so, even if a service returns an HTTP redirect it should not be possible to access the DB since that would require a) including data in the redirected request b) talk something different than HTTP with the MongoDB. But I wouldn't guarantee it. (cc @mauricesvp since I leaked the password used by your service^^)